docs/assistant/cloud/libraries/delete-library.md

```diff
@@ -0,0 +1,19 @@
+---
+id: delete-library
+title: Delete a library
+sidebar_label: Delete a library
+---
+
+## Delete an Assistant Pro Library
+
+You can delete an Assistant Pro library from either the Assistant Pro Cloud or the Assistant Plugin.
+Delete an Assistant Pro Library from The Assistant Pro Cloud
+To delete a library from the Assistant Pro Cloud, select the library you’d like to delete, then from the bottom of the settings window, select Delete This Library.
```

Comment:
To delete a library:
1. In Assistant Pro Cloud, select the library you want to delete.
2. Click **Delete this library** below the settings.

Reply:
Fixed in PR